apps/todo-list-client/src/app/generic-button/generic-button.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-generic-button',
+  templateUrl: './generic-button.component.html',
+})
+export abstract class InputButtonUnitComponent {
```

Comment:
the selector names app-generic-button and your class names InputButtonUnitComponent, please change the name to a appropriate name like GenericButtonComponent